Makefile

```diff
@@ -48,18 +47,42 @@ endif
 export SELF_SIGNED_CERTS   ?= true
 export INSTALLATION_TYPE   ?= managed
 export INSTALLATION_NAME   ?= rhmi
-export INSTALLATION_PREFIX ?= redhat-rhmi
+
 export USE_CLUSTER_STORAGE ?= true
 export OPERATORS_IN_PRODUCT_NAMESPACE ?= false # e2e tests and createInstallationCR() need to be updated when default is changed
 export DELOREAN_PULL_SECRET_NAME ?= integreatly-delorean-pull-secret
 export ALERTING_EMAIL_ADDRESS = noreply-test@rhmi-redhat.com
 
+ifeq ($(INSTALLATION_TYPE), managed)
+	PROJECT=integreatly-operator
+	TAG ?= RHMI_TAG
+	OPERATOR_IMAGE=$(REG)/$(ORG)/$(PROJECT):v$(TAG)
+	NAMESPACE_PREFIX ?= redhat-rhmi-
+	APPLICATION_REPO ?= integreatly
```

Comment:
APPLICATION_REPO confuses me. In the case of managed we are saying it's `integreatly` but in the case of managed-api` we are saying `managed-api-service`. I'm assuming this relates to application in QUAY? Do we use the application `integreatly` for rhmi and the application `managed-api-service`. I think it's probably fine but just want to be sure I'm understanding it correctly. Seems it's only used for operator-courier anyways right?

Reply:
I think this is fine. These values have to do with the update_csv target which you commented about how you still use. This will set the defaults that should be used if its ran for the ORG of integreatly. Which I don't think it every will be.